src/app/destinyTrackerApi/d2-reviewsFetcher.ts

```diff
@@ -165,7 +166,7 @@ class D2ReviewsFetcher {
    */
   getItemReviews(item: DimItem, platformSelection: number) {
     if (!item.reviewable) {
-      return;
+      return $q.when();
```

Comment:
Should this method have been marked `async` as part of the changes here?

Reply:
It could, but then it wouldn’t need these. Sadly there’s some subtleties with async methods vs. angular’s promises that i want to be careful about.